isle-sofa-boot-starter/src/main/java/com/alipay/sofa/isle/spring/listener/SofaModuleContextRefreshedListener.java

```diff
@@ -61,6 +60,27 @@ public void onApplicationEvent(ContextRefreshedEvent event) {
         }
     }
 
+    private List<PipelineStage> getPipelineStagesInApplicationContext(ApplicationContext applicationContext) {
+        Map<String, PipelineStage> beanName2PipelineStage = applicationContext
+            .getBeansOfType(PipelineStage.class);
+        Map<String, PipelineStage> pipelineName2PipelineStage = new HashMap<>();
+
+        for (Map.Entry<String, PipelineStage> beanName2PipelineStageEntry : beanName2PipelineStage
+            .entrySet()) {
+            String pipelineName = beanName2PipelineStageEntry.getValue().getName();
+            PipelineStage oldValue = pipelineName2PipelineStage.get(pipelineName);
+            if (oldValue == null
+                || oldValue.getPriority() < beanName2PipelineStageEntry.getValue().getPriority()) {
+                pipelineName2PipelineStage
+                    .put(pipelineName, beanName2PipelineStageEntry.getValue());
+            }
+        }
+
+        List<PipelineStage> pipelineStages = new ArrayList<>(pipelineName2PipelineStage.values());
+        OrderComparator.sort(pipelineStages);
```

Comment:
The OrderComparator works on PipelineStage , should we move Ordered to PipelineStage instead of AbstractPipelineStage?

Reply:
class not implement Ordered interface can also use OrderComparator.